src/styles/main.scss

```diff
@@ -1,3 +1,14 @@
-body {
+@import "./utils/variables";
+
+@import "./blocks/header";
+@import "./blocks/product_card";
+@import "./blocks/stars";
+@import "./blocks/catalog";
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.reset {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
```

Comment:
i would not use a reset style
create a reset file or even use already written
https://meyerweb.com/eric/tools/css/reset/